docs/how-arbitrum-works/05-validation-and-proving/02-rollup-protocol.mdx

```diff
@@ -0,0 +1,272 @@
+
+Validators can adopt different roles:
+
+1. **Active validators**: Regularly propose new assertions.
+2. **Defensive validators**: Monitor the network and challenge incorrect assertions.
+3. **Watchtower validators**: Passively observe and raise alarms when fraud is detected.
+
+The protocol design requires only one honest validator to secure the system, making Arbitrum trustless and resistant to Sybil attacks.
+
+### Staking mechanism
+
+Some validators will act as bonders at any given time, while others remain passive. Bonders deposit `ETH` bonds into Arbitrum's smart contracts, which are forfeited if they lose a challenge.
+
+:::note
+
+Nitro chains exclusively accept `ETH` as collateral for staking.
+
+:::
+
+A single bond can secure a sequence of assertions, meaning a validator's bond applies to multiple checkpoints of the chain's history. This checkpoint allows efficient resource use while maintaining security.
+
+A validator must be bonded to its predecessor to create a new assertion. The bond ensures that validators have economic risk in any assertion they make.
+
+### Staking rules
+
+1. **New validators**: If unbonded, a validator can bond on the latest confirmed assertion by depositing the required `ETH` bond.
+2. **Extending an existing bond**: The validator can extend its bond to one successor assertion if already bonded. - If a validator submits a new assertion, they automatically extend their existing bond to cover it.
+3. **Unbonding**: A validator can only request a refund if they have a bond on the latest confirmed assertion.
+4. **Losing a challenge**: If a validator's assertion is challenged successfully (i.e., they are malicious), they lose **all** their bonded `ETH` across any unresolved assertions.
+
+Since unbonding is not allowed mid-assertion validators must commit until their assertion is confirmed or disproven.
+
+### Handling disputes and delays
+
+Multiple disputes may be active simultaneously if conflicting assertions arise in the Assertion Tree. However, Arbitrum's protocol ensures that:
+
+- **Honest validators can continue asserting**, building on the last correct assertion.
+- **Users can keep transacting** on the child chain without disruption.
+- **Child-to-parent chain withdrawals** may experience delays - Typically, withdrawals experience a single challenge period (6.4 days) delay. - A key property of BoLD is that we can guarantee that in the common case, withdrawals/messages will only experience delay of one challenge period. In the case of an actual dispute, the withdrawals/messages will be delayed by no more than two challenge periods, regardless of the adversaries' behavior during the challenge.
+
+Despite these delays, Arbitrum guarantees that honest assertions always succeed, maintaining Ethereum-level security.
+
+### Who becomes a validator?
+
+Arbitrum's validation process is permissionless, allowing anyone to participate. However, in practice, most users will not act as validators.
+
```

Comment:
Should we specify that in some Arbitrum chains, the validation process is  not always permissionless?